src/basic/ml/FStar_BigInt.ml

```diff
@@ -0,0 +1,34 @@
+type bigint = Z.t
+type t = bigint
+
+let zero = Z.zero
+let one = Z.one
+let two = Z.of_string "2"
+
+let succ_big_int = Z.succ
+let pred_big_int = Z.pred
+let minus_big_int = Z.neg
+let abs_big_int = Z.abs
+
+let add_big_int = Z.add
+let mult_big_int = Z.mul
+let sub_big_int = Z.sub
+let div_big_int = Z.div
+let mod_big_int = Z.rem
+
+let eq_big_int = Z.equal
+let le_big_int = Z.leq
+let lt_big_int = Z.lt
+let ge_big_int = Z.geq
+let gt_big_int = Z.gt
+
+let square_big_int = Z.sqrt
```

Comment:
this looks like a typo too

Reply:
Or at least a poorly named function

Reply:
Changing the interface to expose `sqrt_big_int` since `square_big_int` can anyway be coded up by a client using `mult_big_int`.

Reply:
Good catch, I've renamed it to `sqrt_big_int`

Reply:
Good Catch! I've renamed it to `sqrt_big_int`.